discovery_engine/lib/discovery_engine.dart

```diff
@@ -23,7 +23,7 @@ import 'package:xayn_discovery_engine/src/logger.dart' show initLogger, logger;
 export 'package:xayn_discovery_engine/src/api/api.dart';
 export 'package:xayn_discovery_engine/src/discovery_engine_base.dart';
 export 'package:xayn_discovery_engine/src/domain/assets/assets.dart'
-    show Manifest, kAssetsPath;
+    show Manifest, kAssetsPath, tmpFileExt;
```

Comment:
Why we need to expose `tmpfileExt`? (And why also `kAssetsPath` it seems it should be used only internally)

Reply:
it is used by the flutter plugin's `AssetCopier`